src/Forms/GridField/GridFieldAddExistingAutocompleter.php

```diff
@@ -12,6 +12,7 @@
 use SilverStripe\ORM\SS_List;
 use SilverStripe\ORM\DataObject;
 use SilverStripe\ORM\DataList;
+use SilverStripe\ORM\Filters\SearchFilter;
```

Comment:
This import is already define on line 19.

```suggestion
```

Reply:
Removed

Reply:
Removed